packages/devtools_app/lib/src/main.dart

```diff
@@ -80,6 +82,15 @@ class HtmlPerfToolFramework extends HtmlFramework {
           .open('https://github.com/flutter/devtools/issues', '_feedback');
     });
 
+    if (enableFlutterWebTesting) {
+      // Listen for clicks on the 'Try DevTools on Flutter Web' button.
+      queryId('try-flutter-web-devtools')
+        ..hidden = false
+        ..onClick.listen((_) {
+          // TODO(kenz): launch flutter web app.
```

Comment:
As of https://github.com/flutter/devtools/pull/1436, we can do this with navigation to `/flutter.html`.